src/components/comum/comboboxRaca/tipos.js

```diff
@@ -0,0 +1,4 @@
+export default Object.freeze({
```

Comment:
Esses tipos de raça serão usados somente dentro do combobox? Se não podemos mover para uma outra pasta melhor localizada.

Reply:
Hj já sabemos que ele está só em um lugar, por isso eu tb sou a favor de mover ele para onde está sendo usado. 
Se no futuro ele for usado em outros lugares aí ele pode ficar na  pasta comum.